streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -4954,8 +4910,6 @@ public void shouldConvertStandbyTaskToActiveTask() {
         when(activeTaskCreator.createActiveTaskFromStandby(Mockito.eq(standbyTask), Mockito.eq(taskId00Partitions), any()))
             .thenReturn(activeTask);
 
-        replay(consumer);
```

Comment:
I assume that consumer is never used in these tests where we have removed the replay?

In that case, should we do something similar to what you did in shouldReturnFalseWhenThereAreStillNonRunningTasks , i.e. test for no interactions